ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsProcess.h

```diff
@@ -0,0 +1,144 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include "ProcessLib/Process.h"
+#include "ThermoHydroMechanicsProcessData.h"
+#include "LocalAssemblerInterface.h"
+
+namespace ProcessLib
+{
+namespace ThermoHydroMechanics
+{
+struct LocalAssemblerInterface;
+
+/// Linear kinematics poro-mechanical/biphasic (fluid-solid mixture) model.
+///
+/// The mixture momentum balance and the mixture mass balance are solved under
+/// fully saturated conditions.
```

Comment:
Are there assumptions for the temperature implemented in the process that should be mentioned here?

Reply:
I have added the description of the thermal part into it.